### PR TITLE
Better netteForms.js

### PR DESCRIPTION
**Pull request by jahudka**
- bugfix: adding event listerers using the standard interfaces
- feature: initialisation on load can be disabled by defining a global Nette object with the 'noInit' property set to true prior to loading the netteForms.js asset

### CODE REVIEW

File: src/assets/netteForms.js [hunk 1]
```diff
@@ -27,17 +33,27 @@
...
-		return callback.apply(element, arguments);
-	};
+	if ('addEventListener' in element) {
+		element.addEventListener(on, callback, false);
+
+	} else if ('attachEvent' in element) {
+		element.attachEvent('on' + on, getHandler(callback));
+
+	} else {
+		throw new Error('NetteForms: Unsupported platform, unable to add event listeners');
```
Comment (dg) [inline src/assets/netteForms.js]: I think this 3rd condition is unnecessary, even jQuery doesn't count with it.

**Comment by dg** [comment]
This is not 'bugfix', I'll change commit message.

File: src/assets/netteForms.js [hunk 2]
```diff
@@ -27,17 +33,27 @@
 
 var Nette = {};
 
+function getHandler(callback) {
+	return function(evt) {
+		callback.call(this, evt || window.event);
```
Comment (dg) [inline src/assets/netteForms.js]: Couldn't this break https://github.com/jahudka/forms/blob/dca62a0029af4b558aa1c7572968fc274a5dc11d/src/assets/netteForms.js#L580 ?

Comment (jahudka) [inline src/assets/netteForms.js]: Ah, true; didn't know IE didn't support `evt.target` until version 9.. Can be fixed at the location you linked though, if you replace `var target = e ? e.target : event.srcElement;` with `var target = e.target || e.srcElement;`.

**Comment by dg** [comment]
First part merged https://github.com/nette/forms/commit/fcb828969fb958d855b152d4072d351257edface

**Comment by dg** [comment]
97a6638a2cacde72e4fd1f6cf8e98828b47196cd

**Pull request closed without merging**